src/s3/putobject.cpp

```diff
@@ -29,15 +29,51 @@ asio::awaitable<void> irods::s3::actions::handle_putobject(
     boost::beast::error_code ec;
     connection_handle connection = get_connection();
 
-    std::cout << "Hi! About to auth" << std::endl;
+    std::cout << " ********* HEADERS *********" << std::endl;
+    for (const auto& field : parser.get()) {
+        std::cout << "HEADER: " << field.name_string() << ":" << field.value() << std::endl;
+    }
+    std::cout << " ***************************" << std::endl;
```

Comment:
please remove any debugging - or convert to 'debug' or 'trace', as appropriate.

Reply:
I removed these lines for now.  Once we get a proper logger I may want to add them back in with debug mode.